src/app/libp2p_helper/src/codanet.go

```diff
@@ -243,6 +264,10 @@ func (gs *CodaGatingState) InterceptAccept(addrs network.ConnMultiaddrs) (allow
 		gs.logGate()
 	}
 
+	if allow {
```

Comment:
can you add a comment here explaining why it's correct to MarkPrivateAddr as known here